make error

when trying to make an up-to-date release
```
make release -j2
Cloning into 'FastCarPlay'...
remote: Enumerating objects: 377, done.
remote: Counting objects: 100% (132/132), done.
remote: Compressing objects: 100% (86/86), done.
remote: Total 377 (delta 87), reused 80 (delta 46), pack-reused 245 (from 1)
Receiving objects: 100% (377/377), 340.62 KiB | 701.00 KiB/s, done.
Resolving deltas: 100% (240/240), done.
Clean complete
xxd -i -n background src/resource/background.bmp > src/autogen/background.bmp.cpp
xxd -i -n font src/resource/font.ttf > src/autogen/font.ttf.cpp
make BUILD_TYPE=release TARGET=./out/app CXXFLAGS="-Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG" LDFLAGS="-Ofast -march=native -Wl,--gc-sections -flto" build
make[1]: Entering directory '/home/carplay/FastCarPlay'
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/pcm_audio.cpp -o out/release/pcm_audio.o
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/settings.cpp -o out/release/settings.o
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/recorder.cpp -o out/release/recorder.o
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/interface.cpp -o out/release/interface.o
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/decoder.cpp -o out/release/decoder.o
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/pipe_listener.cpp -o out/release/pipe_listener.o
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/protocol.cpp -o out/release/protocol.o
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/aes_cipher.cpp -o out/release/aes_cipher.o
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/autogen/background.bmp.cpp -o out/release/autogen/background.bmp.o
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/autogen/font.ttf.cpp -o out/release/autogen/font.ttf.o
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/main.cpp -o out/release/main.o
g++ -Wall -Isrc -Ofast -march=native -fno-plt -fno-rtti -flto -fdata-sections -ffunction-sections -DNDEBUG -c src/renderer.cpp -o out/release/renderer.o
src/main.cpp:337:6: error: ‘cout’ in namespace ‘std’ does not name a type
  337 | std::cout << "[Main] Stopping" << std::endl;
      |      ^~~~
In file included from src/main.cpp:5:
/usr/include/c++/14/iostream:63:18: note: ‘std::cout’ declared here
   63 |   extern ostream cout;          ///< Linked to standard output
      |                  ^~~~
src/main.cpp:338:15: error: expected constructor, destructor, or type conversion before ‘(’ token
  338 | SDL_HideWindow(window);
      |               ^
src/main.cpp:339:1: error: expected declaration before ‘}’ token
  339 | }
      | ^
make[1]: *** [Makefile:56: out/release/main.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/home/carplay/FastCarPlay'
make: *** [Makefile:41: prepare] Error 2
```

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -333,9 +333,9 @@ void application()
             frameStart += 1;
         }
     }
-}
-std::cout << "[Main] Stopping" << std::endl;
-SDL_HideWindow(window);
+
+    std::cout << "[Main] Stopping" << std::endl;
+    SDL_HideWindow(window);
 }
 
 bool setAudioDriver()

